ql/src/java/org/apache/hadoop/hive/ql/parse/HiveParser.g

```diff
@@ -2366,7 +2366,7 @@ columnNameOrder
 @after { popMsg(state); }
     : identifier orderSpec=orderSpecification? nullSpec=nullOrdering?
     -> {$orderSpec.tree == null && $nullSpec.tree == null}?
-            ^(TOK_TABSORTCOLNAMEASC ^(TOK_NULLS_FIRST identifier))
+            ^(TOK_TABSORTCOLNAMEASC ^(TOK_NULLS_LAST identifier))
```

Comment:
Can we add a property to HiveConf.java and set this value depending on that property (default value will be ```NULLS_LAST```)?
In ```HiveLexer.g``` you can find an example on how it is done for quotation, it can be similar to that.

Reply:
Thanks. I made it configurable on 7th patch.